syntax = "proto3";

import "hq/dto/Security.proto";
import "hq/dto/Broker.proto";
option java_package = "com.huasheng.quant.open.sdk.protobuf.hq.response";
option java_outer_classname = "BrokerResponseProto";
message BrokerResponse {
    Security security = 1; //股票定义
    repeated Broker brokerAskList = 2; //经纪Ask(卖)盘
	repeated Broker brokerBidList = 3; ////经纪Bid(买)盘
}
